config/locales/models/activity.en.yml

```diff
@@ -46,7 +46,7 @@ en:
         planned_end_date: For example, 28 11 2020
         planned_start_date: For example, 27 3 2020
         sector_category: What area of the economy or society is your %{level} helping? For example, research, education or small to medium-sized enterprise (SME) development. Choose one of the <a href='https://www.oecd.org/dac/stats/documentupload/2015%20CRS%20purpose%20codes%20EN_updated%20April%202016.pdf' target='_blank' class='govuk-link'>CRS purpose codes</a>
-        title: A short title that explains the fund's activity
+        title: A short title that explains the purpose of the programme
```

Comment:
The activity level here has been changed from fund to programme. I actually think it should be `%{level}` so the activity type name can change with the level.